src/language-js/print/call-arguments.js

```diff
@@ -159,7 +159,7 @@ function printCallArguments(path, options, print) {
       shouldBreakForArrowFunction;
 
     // We want to print the last argument with a special flag
-    let printedExpanded;
+    let printedExpanded = [];
```

Comment:
Are we sure `printedExpanded` is no longer empty by the time `printedExpanded[0]` is accessed by the code below?

Reply:
Yes, https://github.com/prettier/prettier/blob/a66e43a09319bf13beda421d00c04c6228a4ccba/src/language-js/print/call-arguments.js#L38